README.md

```diff
@@ -5,31 +5,40 @@
 **running this library with python version 3.8 will result in error** 
 
 ## Installing requirements: 
-- pip install -r requirements.txt
+```shell
+pip install -r requirements.txt
+```
 
-## run sampleAcquis.py example : 
+## run `sampleAcquis.py` example: 
+```
```

Comment:
Would the `shell` keyword make sense here, too? All the other code-blocks use it.

Reply:
I assume removing the `shell` keyword in this commit is an artifact of rebasing?

Reply:
yeah, I managed to do it properly now